example/basic/index.js

```diff
@@ -1,3 +1,4 @@
+'use strict'
```

Comment:
Cool, we should just remove it consistently from all source files then? I was actually wondering what we should do on a project wide basis, since the linter doesn't care.